src/Microsoft.Sbom.Common/ComplianceStandard/ComplianceExtensions.cs

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.Sbom.Common.ComplianceStandard;
+
+internal static class ComplianceExtensions
+{
+    /// <summary>
+    /// Gets the common entity type that is used by the parser.
+    /// </summary>
+    internal static string GetCommonEntityType(string entityType)
```

Comment:
This can be an actual extension method
```suggestion
    internal static string GetCommonEntityType(this string entityType)
```